gen-phpweb-sqlite-db.php

```diff
@@ -179,14 +179,11 @@ function scan_langs($root)
 	if (!$d) {
 		return;
 	}
-	readdir($d); readdir($d);
 	while (($f = readdir($d)) !== false) {
-		if ($f === '.svn') {
-			continue;
-		}
-		if ($f === '.git') {
+		if (strpos($f, '.') !== false) {
```

Comment:
Shouldn't this be simply `if ($f[0] === '.')`, a `.` could exist in the reset of the filename.

Reply:
We could do this, however a dot is not a valid character in a language code.